include/fastdds/dds/publisher/DataWriter.hpp

```diff
@@ -101,9 +99,7 @@ class DataWriter : public DomainEntity
      * Write data with params to the topic.
      * @param data Pointer to the data
      * @param params Extra write parameters.
-     * @return True if correct
-     * @par Calling example:
-     * @snippet fastrtps_example.cpp ex_PublisherWrite
+     * @return True if correct, false otherwise
      */
     RTPS_DllAPI bool write(
             void* data,
```

Comment:
Should we change the return type of this to `ReturnCode_t` (perhaps on a different PR)?